test/api/controllers/validators/validations/BooleanValidationSpec.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package api.controllers.validators.validations
+
+import api.models.errors.BadRequestError
+import api.models.utils.JsonErrorValidators
+import support.UnitSpec
+
+class BooleanValidationSpec extends UnitSpec with JsonErrorValidators {
+
+  "validate" should {
+    "return no errors" when {
+      "when the string true is supplied" in {
+        val validBoolean     = "true"
+        val validationResult = BooleanValidation.validate(validBoolean, BadRequestError)
+        validationResult shouldBe Nil
+      }
+
+      "when the upper case string TRUE is supplied" in {
+        val validBoolean     = "TRUE"
+        val validationResult = BooleanValidation.validate(validBoolean, BadRequestError)
```

Comment:
Just for consistency, can these all be "result" instead of "validationResult"? Then the tests all follow the same kind of format